release-please-config.json

```diff
@@ -115,6 +115,14 @@
       "bump-patch-for-minor-pre-major": true,
       "versioning": "default",
       "extra-files": []
+    },
+    "providers/go-feature-flag-in-process": {
```

Comment:
Is this supposed to be included in this PR?

Reply:
Actually not, I will remove it